wasd-controls.js

```diff
@@ -8,6 +8,8 @@ export class WasdControlsComponent extends Component {
     static Properties = {
         /** Movement speed in m/s. */
         speed: {type: Type.Float, default: 0.1},
+        /** Flag for only moving the object on the global x & z planes */
+        alignedToFloor: {type: Type.Bool, default: false},
```

Comment:
I would probably call that `lockY` / `lockHeight` or something, because we could imagine that there is a component running just after this one that computes the height (could be from a terrain vertex, from a physx engine, anything)